.github/workflows/main.yml

```diff
@@ -0,0 +1,55 @@
+name: Build DAPHNE
+
+on:
+  workflow_dispatch:
+  push:
+
+jobs:
+  build:
+    runs-on: ubuntu-22.04
+    env:
+      # fix usage of tput in build script:
+      TERM: dumb
+      DEBIAN_FRONTEND: noninteractive
+
+    steps:
+    -
+      name: "Install build dependencies"
+      run: |
+        set -e
+        pwd
+        echo Installing ...
+        sudo apt-get update
+        sudo apt-get install -y cmake clang default-jdk git libssl-dev lld pkg-config python3 python3-pip gfortran uuid-dev libboost-dev unzip wget moreutils
+        sudo wget -qO /usr/local/bin/ninja.gz https://github.com/ninja-build/ninja/releases/latest/download/ninja-linux.zip
+        sudo gunzip /usr/local/bin/ninja.gz
+        sudo chmod a+x /usr/local/bin/ninja
```

Comment:
What's wrong with the packaged version of ninja?

Reply:
hm, I could not found a packaged version in apt nor snap ??

Reply:
It's called "ninja-build" ;-)